node/src/external/wrapper/node_wrapper.h

```diff
@@ -0,0 +1,271 @@
+// See LICENSE file
+
+#ifndef SRC_JS_HEADER_
+#define SRC_JS_HEADER_
```

Comment:
How are we going to keep this in sync with the version in node-apphost repo ? I am still unclear (rather unfamiliar) on how, in general, libs/files/binaries across repos referenced.